### DISCUSSION TITLE
如果节点IP变更了，我该怎么处理？

### DISCUSSION THREAD

**Discussion by codering** [pgsty/pigsty#470] [Q&A]
我目前是在单节点部署，现在这个节点ip变了， 我是不是需要按照“三步走”重新安装？ 节点配置较低，我看没有开启自动备份，那我在重新安装之前需要自己手动备份数据？

**Comment by Vonng** [comment]
如果你用单节点，投机取巧的办法是使用 127.0.0.1 作为 IP 地址。
